build-system/pr-check/utils.js

```diff
@@ -160,6 +161,9 @@ function downloadOutput_(functionName, outputFileName) {
       `${fileLogPrefix} Downloading build output from ` +
       colors.cyan(buildOutputDownloadUrl) + '...');
   exec('echo travis_fold:start:download_results && echo');
+  decryptTravisKey_();
+  execOrDie(`gsutil signurl -d 3m ${OUTPUT_STORAGE_KEY_FILE} ` +
+      `${OUTPUT_STORAGE_LOCATION}/${outputFileName}`);
```

Comment:
Glad you got this to work! (During my failed attempt, I was trying [`gcloud auth`](https://cloud.google.com/sdk/gcloud/reference/auth/) instead of `gsutil signurl`.)

Reply:
J